### PR TITLE
Naming convention update

### PR DESCRIPTION
**Pull request by afloresOSI**
Greg, this PR just updates file and folder names to bring EDS inline with the standards needed for the new documentation site. The only content changes fix links. Nothing for you to review.

### CODE REVIEW

**Review by gaholt — APPROVED** [review]

File: V1/Configuration/toc.yml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 - name: Data ingress configuration
-  href: ../DataIngress/toc.yml
-  topicHref: ../DataIngress/EDSDataIngress_1-0.md
+  href: ../data-ingress/toc.yml
+  topicHref: ../data-ingress/eds-data-ingress.md
 - name: Storage
-  href: Storage/toc.yml
-  topicHref: Storage_1-0.md
+  href: storage/toc.yml
+  topicHref: Storage.md
```
Comment (TinaUll) [inline V1/Configuration/toc.yml:11]:
```suggestion
  topicHref: storage.md
```

File: V1/Overview/toc.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 - name: Design considerations
-  href: ../Design/ScalePerformance_1-0.md
+  href: scale-performance.md
```
Comment (TinaUll) [inline V1/Overview/toc.yml:2]:
```suggestion
  href: ../design/scale-performance.md
```

File: V1/index.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ EDS requires an endpoint to connect to REST APIs on the local device, which is s
 
 Orange arrows show data flowing into EDS and blue arrows show data flowing out of EDS.
 
-For detailed information about configuring each component of EDS, see [Configuration](xref:Configuration1-0)
+For detailed information about configuring each component of EDS, see [Configuration](xref:Configuration).
```
Comment (TinaUll) [inline V1/index.md:45]: Great job, getting rid of those version numbers :)

File: v1/egress/configure-data-egress.md [hunk 4]
```diff
@@ -35,9 +35,9 @@ curl -d "@PeriodicEgressEndpoints.config.json" -H "Content-Type: application/jso
...
 | **ClientSecret**                | Required for OCS endpoint | string    | Used for authentication with the OCS OMF endpoint. |
-| **DebugExpiration**             | Optional                  | string    | Enables logging of detailed information, for each outbound HTTP request pertaining to this egress endpoint, to disk. The value represents the date and time this detailed information should stop being saved. Examples of valid strings representing date and time:  UTC: "yyyy-mm-ddThh:mm:ssZ", Local: "mm-dd-yyyy hh:mm:ss". For more information, see [Troubleshoot Edge Data Store](../Troubleshooting/Troubleshooting_1-0.md). |
+| **DebugExpiration**             | Optional                  | string    | Enables logging of detailed information, for each outbound HTTP request pertaining to this egress endpoint, to disk. The value represents the date and time this detailed information should stop being saved. Examples of valid strings representing date and time:  UTC: "yyyy-mm-ddThh:mm:ssZ", Local: "mm-dd-yyyy hh:mm:ss". For more information, see [Troubleshoot Edge Data Store](../Troubleshooting/Troubleshooting.md). |
```
Comment (TinaUll) [inline v1/egress/configure-data-egress.md:38]: Wouldn't that use a UID instead of the file path?

File: v1/egress/configure-data-egress.md [hunk 5]
```diff
@@ -35,9 +35,9 @@ curl -d "@PeriodicEgressEndpoints.config.json" -H "Content-Type: application/jso
...
 | **Description**                 | Optional                  | string    | Friendly description |
-| **EgressFilter**                | Optional                  | string    | A filter used to determine which streams and types are egressed. For more information on valid filters, see [Search in SDS](../Sds/Search_1-0.md). |
+| **EgressFilter**                | Optional                  | string    | A filter used to determine which streams and types are egressed. For more information on valid filters, see [Search in SDS](../Sds/Search.md). |
```
Comment (TinaUll) [inline v1/egress/configure-data-egress.md:40]: Same here?

File: v1/overview/command-line-linux-quick-start.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
 ---
-uid: commandLineLinuxQuickStart1-0
+uid: commandLineLinuxQuickStart
 ---
 
 # Command line quick start - Linux
 
-The EdgeCmd utility is OSIsoft's proprietary tool for configuring Edge Data Store from a command line. EdgeCmd must be installed on the device with Edge Data Store. For instructions on installing EdgeCmd, see the [EdgeCmd utility help](https://osisoft.github.io/Edgecmd-Docs/V1.0/EdgeCmd_utility_1-0.html).
+The EdgeCmd utility is OSIsoft's proprietary tool for configuring Edge Data Store from a command line. EdgeCmd must be installed on the device with Edge Data Store. For instructions on installing EdgeCmd, see the [EdgeCmd utility help](https://osisoft.github.io/Edgecmd-Docs/v1.0/EdgeCmd_utility.html).
```
Comment (TinaUll) [inline v1/overview/command-line-linux-quick-start.md:7]: I don't think EdgeCmd_utility.html exists (anymore). Sounds like it should be index.html instead

File: v1/overview/command-line-quick-start-windows.md [hunk 7]
```diff
@@ -1,10 +1,10 @@
 ---
-uid: commandLineWindowsQuickStart1-0
+uid: commandLineWindowsQuickStart
 ---
 
 # Command line quick start - Windows
 
-The EdgeCmd utility is OSIsoft's proprietary tool for configuring Edge Data Store from a command line. EdgeCmd must be installed on the device with Edge Data Store. For instructions on installing EdgeCmd, see the [EdgeCmd utility help](https://osisoft.github.io/Edgecmd-Docs/V1.0/EdgeCmd_utility_1-0.html).
+The EdgeCmd utility is OSIsoft's proprietary tool for configuring Edge Data Store from a command line. EdgeCmd must be installed on the device with Edge Data Store. For instructions on installing EdgeCmd, see the [EdgeCmd utility help](https://osisoft.github.io/Edgecmd-Docs/v1.0/EdgeCmd_utility.html).
```
Comment (TinaUll) [inline v1/overview/command-line-quick-start-windows.md:7]: Same here

```suggestion
The EdgeCmd utility is OSIsoft's proprietary tool for configuring Edge Data Store from a command line. EdgeCmd must be installed on the device with Edge Data Store. For instructions on installing EdgeCmd, see the [EdgeCmd utility help](https://osisoft.github.io/Edgecmd-Docs/v1.0/index.html).
```

File: z-archive/Reference/toc.yml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+- name: Sequential Data Store (SDS)
+  href: ../SDS/toc.yml
```
Comment (TinaUll) [inline z-archive/Reference/toc.yml:2]: I assume you did not change this one because it is Archive?

**Comment by TinaUll** [comment]
@afloresOSI I just had the few comments - great job!

File: z-archive/Reference/toc.yml (hunk 8 above)
Comment (afloresOSI) [reply z-archive/Reference/toc.yml:2]: Yes, that is material that is no longer used in the help, which is why it is outside the v1 folder. I just haven't had the heart to delete it.

File: v1/egress/configure-data-egress.md (hunk 4 above)
Comment (afloresOSI) [reply v1/egress/configure-data-egress.md]: Updated to UID

File: v1/egress/configure-data-egress.md (hunk 5 above)
Comment (afloresOSI) [reply v1/egress/configure-data-egress.md]: Updated to UID.

**Pull request merged**